components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -141,11 +139,11 @@ void SHOCMacrophysics::initialize_impl (const util::TimeStamp& t0)
   auto qc               = m_shoc_fields_out["qc"].get_reshaped_view<Spack**>();
   auto qv               = m_shoc_fields_out["qv"].get_reshaped_view<Spack**>();
   auto tke              = m_shoc_fields_out["tke"].get_reshaped_view<Spack**>();
-  auto s                = m_shoc_fields_out["s"].get_reshaped_view<Spack**>();
   auto Q                = m_shoc_fields_out["Q"].get_reshaped_view<Spack***>();
   auto cldfrac_liq      = m_shoc_fields_out["cldfrac_liq"].get_reshaped_view<Spack**>();
   auto sgs_buoy_flux    = m_shoc_fields_out["sgs_buoy_flux"].get_reshaped_view<Spack**>();
   auto inv_qc_relvar    = m_shoc_fields_out["inv_qc_relvar"].get_reshaped_view<Spack**>();
+  auto phis             = m_shoc_fields_in["phis"].get_reshaped_view<const Pack1d*>();
```

Comment:
This might be a bit more general than just this PR: I noticed shoc uses Pack1d=Pack<Scalar,1>; What's the point in using this vs just Scalar?

Reply:
Yes, this is all over SHOC. Seemed strange to me when I started working on shoc conversion, but I never asked about it.. I can create a separate PR changing these in shoc.